src/Pack.js

```diff
@@ -0,0 +1,194 @@
+/**
+    @external Viz
+    @see https://github.com/d3plus/d3plus-viz#Viz
+*/
+import {nest} from "d3-collection";
+import {hierarchy, pack} from "d3-hierarchy";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import * as shapes from "d3plus-shape";
+import {Viz} from "d3plus-viz";
+
+/**
+    @class Pack
+    @extends Viz
+    @desc Uses the [d3 pack layout](https://github.com/d3/d3-hierarchy#pack) to creates Circle Packing chart based on an array of data.
+*/
+export default class Pack extends Viz {
+
+  /**
+      @memberof Pack
+      @desc Invoked when creating a new class instance, and sets any default parameters.
+      @private
+  */
+  constructor() {
+
+    super();
+
+    this._layoutPadding = 1;
+    this._on.mouseenter = () => {};
+    this._on["mouseleave.shape"] = () => {
+      this._tooltip = false;
+      this.hover(false);
+    };
```

Comment:
Why are you overriding the default mouseleave? This is no different from the default behavior.

Reply:
@davelandry because if I move the mouse from leaf circle (with tooltip) to another level in packing, the tooltip position is frozen.
![image](https://user-images.githubusercontent.com/5233593/42709121-6e738fb6-86ad-11e8-9642-672ded081d89.png)

Reply:
never mind, I solved it! :)